examples/todomvc/src/lib.rs

```diff
@@ -230,12 +242,14 @@ impl Model {
             html! {
                 <input class="edit"
                        type="text"
+                       ref= self.focus_ref.clone()
                        value=&self.state.edit_value
+                       onmouseover=self.link.callback(|_| Msg::Focus)
                        oninput=self.link.callback(|e: InputData| Msg::UpdateEdit(e.value))
                        onblur=self.link.callback(move |_| Msg::Edit(idx))
                        onkeypress=self.link.callback(move |e: KeyboardEvent| {
                           if e.key() == "Enter" { Msg::Edit(idx) } else { Msg::Nope }
-                       }) />
+                       })/>
```

Comment:
format: the space is desired
```suggestion
                       }) />
```